cinch/site.yml

```diff
@@ -31,6 +31,17 @@
   roles:
     - repositories
 
+- name: enable additional package streams on RHEL 8
```

Comment:
This whole play needs to be converted into a role. Now would be a good time to test integrating Cinch with roles from the Oasis project.

Reply:
You are welcome to submit  a PR.

Reply:
I can definitely submit PRs for switching to Oasis roles as well (I need to carve out time to work with the Ops team in testing how it works with Ansible Tower). In the meantime, this code should not live in `site.yml` as part of this PR but should be in a role inside of Cinch.